src/components/App/App.component.js

```diff
@@ -1,9 +1,14 @@
 // @flow
 /* eslint-disable */
-import React, {Component} from 'react';
+import './app.css';
+import * as React from 'react';
```

Comment:
Isn't this the same as `imoprt React from 'react'`?

Reply:
flow types are also imported when using * as React (among other stuff I guess)